### DISCUSSION TITLE
Does socket-io python supports webtransport ?

### DISCUSSION THREAD

**Discussion by ViniciusBelloli** [miguelgrinberg/python-socketio#1249] [General]
I've been checking that the node.js version already supports the webtransport protocol and I'd like to know if the python library already has this functionality, or if there is any intention of implementing it.

**Comment by miguelgrinberg** [comment]
There is no support for WebTransport at this time. I'm actually not aware of any WebTransport capable Python web servers, which would be a prerequisite for adding it to this package.

**Comment by ViniciusBelloli** [comment]
Thanks for your answer, I thought maybe aioquic would work.
